src/webui/api/appcontroller.cpp

```diff
@@ -424,6 +430,11 @@ void AppController::preferencesAction()
     data[u"request_queue_size"_s] = session->requestQueueSize();
     // DHT bootstrap nodes
     data[u"dht_bootstrap_nodes"_s] = session->getDHTBootstrapNodes();
+    // I2P
```

Comment:
IMO, there is no point in splitting I2P related stuff in this place of code. I would keep them all together.

Reply:
Moved.